sdk/rust/src/http.rs

```diff
@@ -61,6 +61,30 @@ impl Request {
         builder
     }
 
+    /// Creates a [`RequestBuilder`] to fetch the headers from `uri`
+    pub fn head(uri: impl Into<String>) -> RequestBuilder {
+        RequestBuilder::new(Method::Head, uri)
+    }
+
+    /// Creates a [`RequestBuilder`] to DELETE the resource specified by `uri`
+    pub fn delete(uri: impl Into<String>) -> RequestBuilder {
```

Comment:
Nit: I would put `delete` after `put` and `patch` so that those methods are next to `post`.